openpype/hosts/blender/api/plugin.py

```diff
@@ -88,6 +97,102 @@ def create_blender_context(active: Optional[bpy.types.Object] = None,
     raise Exception("Could not create a custom Blender context.")
 
 
+def create_container(name, color_tag=None):
+    """
+    Create the container with the given name
+    """
+    # search in the container already exists
+    container = bpy.data.collections.get(name)
+    # if container doesn't exist create them
+    if container is None:
+        container = bpy.data.collections.new(name)
+        if color_tag:
+            container.color_tag = color_tag
+        bpy.context.scene.collection.children.link(container)
+        return container
+
+
+def remove_container(container, content_only=False):
+    """
+    Remove the container with all this objects and child collections.
+
```

Comment:
Please document this fuction's arguments

Reply:
done